monero-outreach-docs/translations/fr/archive/rpc_pay_fr.md

```diff
@@ -0,0 +1,133 @@
+
+Le Paiement-RPC est également privé. Il n'y a pas de grand solde de crédits dus qui seront maintenus et qui pourraient nécessiter un partage d'informations pour la redondance ou la sécurité. Les hash de minage utilisé pour le paiement sont anonymes et il n'est pas possible d'en identifier le propriétaire sur la chaîne de blocs. Un identifiant de paiement peut être appliqué pour faire le suivi des soldes, mais la procédure d'opération associée aux très petits montants en jeu permet à l'identifiant d'être régulièrement changé ou, à rigueur, de ne carrément pas être utilisé.
+
+Il est également pertinent de considérer les désavantages de l'implémentation actuelle du Paiement-RPC dans le _monerod_ ou les portefeuilles. Celle-ci peut devenir un fardeau pour les utilisateurs ayant une faible capacité de calcul (p.ex. les appareils mobiles, qui sont dépendants du soutien d'un tiers avec plus de puissance de calcul pour leur hash). De plus, la récompense qui revient au serveur est aléatoire: elle est seulement octroyée lorsqu'un client fournit un hash "gagnant" qui servira à miner un bloc, tandis que les sorties du serveur doivent être continues et doivent fournir une provision régulière de hash. Finalement, les montants qu'un serveur peut générer sont limités par l'échéancier de minage de Monero et ont tendance à être faibles.
+
+Le Paiement-RPC a plus encore à offrir qu'un simple mécanisme de paiement et de récompense entre les différents programmes de Monero, puisque tout serveur peut utiliser le Paiement-RPC pour générer un revenu. Le logiciel du Paiement-RPC, incorporé dans celui du nœud et portefeuille officiel de Monero, est à code source ouvert et peut être réutilisé pour d'autres applications innovantes, ou peut servir à habiliter d'autres applications à communiquer avec les instances monerod. Le concept fondamental demeure que le serveur donne son adresse Monero à ses clients, qui l'utilisent pour créer des hash de minage Monero qui sont ensuite envoyés au serveur en guise de paiement. [L'Exemple #3](https://www.monerooutreach.org/stories/RPC-Pay.html#box3) illustre une portion du code source du logiciel de Paiement-RPC.
+
+### _Primo_
+
+Le Projet Primo représente une nouvelle application du Paiement-RPC ([repo.getmonero.org/selene/primo](https://repo.getmonero.org/selene/primo)). Primo est un protocole et une suite logicielle qui supporte le paiement pour la livraison de contenu web. Il permet à des annonces web indésirables d'être remplacées par des générateurs de hash invisibles et discrets. Quelqu'un qui visite un site web qui roule Primo et qui ne veut pas voir d'annonces peut opter pour la génération de hash de minage Monero.
```

Comment:
qui roule Primo >>> 'qui fait rouler Primo'  OR 'qui fait tourner Primo'  

Reason: Preference (probably due to regional difference between the translator and the reviewer)

Reply:
I actually meant this as a translation of "which runs Primo", rather than "on which Primo runs". "Rouler" may be a somewhat colloquial translation of "run" (like for example, "des scripts qui roulent en arrière-plan"), but I've worked with French Canadian software engineers who use this term. I'll leave this one as-is, if reviewer accepts.

Reply:
Ok, I am fine then. I missed this on the english text.